group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorConfig.java

```diff
@@ -357,6 +378,12 @@ public GroupCoordinatorConfig(AbstractConfig config) {
         require(shareGroupSessionTimeoutMs <= shareGroupMaxSessionTimeoutMs,
             String.format("%s must be less than or equals to %s",
                 SHARE_GROUP_SESSION_TIMEOUT_MS_CONFIG, SHARE_GROUP_MAX_SESSION_TIMEOUT_MS_CONFIG));
+        require(shareGroupRecordLockDurationMs >= shareGroupMinRecordLockDurationMs,
+                String.format("%s must be greater than or equals to %s",
```

Comment:
As @smjn mentioned, indentation. At the least, the indentation should match the surrounding code.